Src/Processor/Standard/StandardScore.cpp

```diff
@@ -83,6 +83,11 @@ void CStandardScore::ComputeTotalValue()
 		multiplier *= 0.95f;
 	}
 
+	if((_mods & EMods::TouchDevice) > 0)
+	{
+		_aimValue = pow(_aimValue, 0.8f);

```

Comment:
The suggestion was to scale aim stars by ^0.8, not the final pp value. That is an important difference. It should be applied to the output of `beatmap.DifficultyAttribute(_mods, CBeatmap::Aim)` in line 101.